docs/_panels/github-repositories.md

```diff
@@ -0,0 +1,21 @@
+---
+title: GitHub Repositories
+description: metrics focused on repositories popularity.
+author: Bitergia
+screenshot: sigils/github-repositories.png
+created_at: 
+grimoirelab_version: 0.2.0
+layout: panel
+---
+
+The GitHub Repositories panel show information about the popularity of a repository
+in GitHub. The popularity is based in the number of forks, times a repository has
+been stared and the number of subscribers.
+
+The panel shows a summary of these indicators showing the maximum number of each
+one per repository.
+
+Popularity indicators have two types of visualizations. Each visualization displays
+the top repositories for that indicator in particular. The first visualization shows
+the growth over time while the second one shows the variation from one period to
+another.
```

Comment:
Please follow the same structure as in https://chaoss.github.io/grimoirelab-sigils/panels/overall-community-structure/ and https://chaoss.github.io/grimoirelab-sigils/panels/jenkins-job-categories/

Basically it contains a description of the panel and the main use case behind it (what we expect the user to look at using the panel, see for instance above's jenkins panel documentation) and a section named `Metrics` with an entry describing each widget.

Reply:
For instance Repositories table is showing maximum values for the selected period of time. This would need a brief explanation like this in the documentation to let users understanding the numbers and knowing what metrics are those and how they work.